prov/verbs/src/verbs_info.c

```diff
@@ -1365,10 +1365,20 @@ static int fi_ibv_get_match_infos(uint32_t version, const char *node,
 		return ret;
 	}
 
-	if (fi_ibv_handle_sock_addr(node, service, flags, hints, info) &&
-	    fi_ibv_handle_ib_ud_addr(node, service, flags, info)) {
+	ret_sock_addr = fi_ibv_handle_sock_addr(node, service, flags, hints, info);
+	if (ret_sock_addr)
+		VERBS_INFO(FI_LOG_CORE, "Handling of the socket address fails - %d\n",
+			   ret_sock_addr);
+	ret_ib_ud_addr = fi_ibv_handle_ib_ud_addr(node, service, flags, info);
+	if (ret_ib_ud_addr)
+		VERBS_INFO(FI_LOG_CORE, "Handling of the IB ID address fails - %d\n",
+			   ret_ib_ud_addr);
```

Comment:
If `fi_ibv_handle_sock_addr` is successful, `fi_ibv_handle_ib_ud_addr` can be skipped, right?

Reply:
Yes, you're right. This can occur in the previous code

Reply:
I mean, wasn't that the expected behavior? My assumption was these functions would be exclusive. fi_ibv_handle_sock_addr handling MSG and RDM EP types and fi_ibv_handle_ib_ud_addr handling just DGRAM EP type.

Reply:
No, it wasn't expected behavior. Since when user passes NODE:SERVICE or dest/src_addr with sockaddr_in structure, we should resolve addresses for MSG, RDM, DGRAM EP types. Then our fi_info items will contain src/dest_addr for MSG, RDM, DGRAM EP types.